AUTHORS.md

```diff
@@ -17,3 +17,4 @@ EasyRPG Player authors
 * scurest
 * Shin-NiL
 * Takeshi Watanabe (takecheeze)
+* Alberto León Meaños (albeleon)
```

Comment:
Please respect the alphabetical order.

You will be the first in the list, congratulation